notebooks/MAST/astroqueryMAST/beginner_program_id_search/beginner_program_id_search.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "<a id=\"top\"></a>\n",
+    "# Accessing JWST Proposal Data in astroquery.mast\n",
+    "## Learning Goals\n",
```

Comment:
Neither the name of the notebook nor the subdirectory path makes it clear this is about finding JWST data by Program ID.  Worth adding "jwst" im the folder and file name somewhere?

Reply:
You're right -- only the title mentions JWST. This notebook could also be useful for people trying to get their HST data, so I wonder if I should leave it where it is, but change the title.